ibis/backends/trino/tests/conftest.py

```diff
@@ -124,7 +124,8 @@ def load_tpch(self) -> None:
                 prefix = prefixes.get(table, table[0])
 
                 t = (
-                    con.table(table, schema=data_schema).rename(f"{prefix}_{{}}".format)
+                    con.table(table, schema=data_schema)
+                    .rename(f"{prefix}_{{}}".format)
```

Comment:
This may be an improvement over Black, as it's more aligned with what the change would be without the interleaved comment.